torch/lib/c10d/ProcessGroupMPI.cpp

```diff
@@ -102,6 +102,25 @@ std::vector<at::Tensor> ProcessGroupMPI::WorkMPI::result() {
   }
 }
 
+c10::intrusive_ptr<c10::ivalue::Future> ProcessGroupMPI::WorkMPI::getFuture() {
+  return future_;
+}
+
+void ProcessGroupMPI::WorkMPI::finishCompleteFuture(std::exception_ptr eptr) {
```

Comment:
Shall we split normal and error cases into two different functions?

Reply:
Done